src/gettext.cpp

```diff
@@ -232,14 +208,73 @@ void init_gettext(const char *path, const std::string &configured_language,
 #endif
```

Comment:
It should be noted that the `_WIN32` check here should be removed since some locales on Linux also do not use a UTF-8 locale by default.